hang_man.py

```diff
@@ -0,0 +1,158 @@
+import os
+import sys
+import copy
+import string
+import random
+import argparse
+import urllib.request
+
+
+class Hangman:
+
+    def __init__(self, min_limit):  # Creating init method that takes in max word limit as perimeter
+        self.min_limit = min_limit  # Minimum word limit taken from user
+        self.word_list = []  # A list that stores all the word from local file or web
+        self.right_list = []  # A list that stores the right word
+        self.guess_letter = str  # User input of their guessed character
+        self.hang_list = [       # A list that contains all the stages of hangman
```

Comment:
You can dry up printing the gallows. Think about the similarities and differences of each variant of the gallows to reduce the amount of code to print all the unique instances.

Reply:
this may also help you keep magic numbers out of your code, which can be hard to track and follow

Reply:
Created a new method called hang_display. Now the program displays status of hangman from that method rather then list.